### ISSUE TITLE
filenames for T1w/T2w ratio images

### ISSUE THREAD

**Issue by loj** [bids-standard/bids-specification#462]
I've been reading BEP003 (#265), and I'm uncertain how to handle the naming of T1w/T2w ratio images (ex: the HCP dataset `<sub>/T1w/T1wDividedByT2w.nii.gz`).

Perhaps `<source_keywords>_desc-dividedByT2w_T1w.nii.gz`?

Any thoughts, opinions, or insights are most welcome. :-)

**Comment by nicholst** [comment]
What about `T1wT2wratio` for the image type?  Concise while hopefully being unambiguous, and it seems like you'd never want to mistakenly put it to use if you're looking for a T1w image.

**Comment by effigies** [comment]
The way I would interpret `*_desc-dividedByT2w_T1w.nii.gz` would be that the image can be used as a T1w map, but I should be aware that "dividedByT2w" happened to it. If that's reasonable, then I think that's fine, and it's certainly valid.

If this measure is not interpretable as a T1w, then it should have its own suffix. I would be fine with @nicholst's suggestion of `T1wT2wratio`. Or is there another (briefer or more meaningful) name you can think of?

**Comment by yarikoptic** [comment]
FWIW, I do like `T1wT2wratio` -- it is somewhat inline with `T1w` as "T1 weighted" - so the additional notion ("weighted") is attached at the end, here `ratio` similarly follows. (Un)fortunately it would never become `T1wT2w-ratio` and `T1wT2w+ratio` kinda would make no sense, so `T1wT2wratio` might be the best (could not come up with shorter and not more confusing)

**Comment by effigies** [comment]
I see no reason not to add this to the derivatives PR. Sorry things are moving so slowly there... I'm aiming to finish up #301 on Monday, which should unblock a lot.

**Comment by yarikoptic** [comment]
so I guess the proper place to introduce it would be a new section within https://github.com/bids-standard/bids-specification/pull/207/files#diff-4ff87c6142082e27a0f84319b01b85ffR89 ?

**Comment by effigies** [comment]
I would say yes to that file, but that section specifically focuses on surface-mapped anatomical scalar derivatives. I think we have two options:

1) Change to "Anatomical scalar derivatives", and include patterns for volumetric NIfTIs, surface-mapped GIFTIs and volume/surface-mapped CIFTI-2s.
2) Add a "Volumetric anatomical scalar derivatives" section.

I would be inclined toward option 1, but we should check in with @edickie and @ahoopes, who wrote this section.

**Comment by edickie** [comment]
Yeah, I think I agree with option 1 - "Anatomical scalar derivatives" that, in this case, are volumetric NifTIs.  Actually, "myelinmap" is already a proposed suffix in this section - but I agree that "T1wT2wratio" might be a better choice of suffix...
